libraries/AP_Relay/AP_Relay.h

```diff
@@ -50,6 +50,9 @@ class AP_Relay {
 
     AP_Int8 _pin[AP_RELAY_NUM_RELAYS];
     AP_Int8 _default;
+    uint8_t _pin_states;
```

Comment:
Do we need to concern ourselves with any startup pin states here?

Reply:
no, since the relay init calls the set routine and updates it...that variable should be 0 at boot, which it is